dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/imports/StringToTrackerIdSchemeParamConverter.java

```diff
@@ -27,22 +27,23 @@
  */
 package org.hisp.dhis.webapi.controller.tracker.imports;
 
+import java.beans.PropertyEditorSupport;
+
 import org.hisp.dhis.common.CodeGenerator;
 import org.hisp.dhis.tracker.TrackerIdScheme;
 import org.hisp.dhis.tracker.TrackerIdSchemeParam;
-import org.springframework.core.convert.converter.Converter;
 import org.springframework.util.StringUtils;
 
-public class StringToTrackerIdSchemeParamConverter implements Converter<String, TrackerIdSchemeParam>
+public class StringToTrackerIdSchemeParamConverter extends PropertyEditorSupport
```

Comment:
Even if we have no convention maybe call it `Editor` instead of `Converter` 🤷🏻

Reply:
Done